picard/pluginmanager.py

```diff
@@ -342,13 +351,15 @@ def _load_plugin(self, name):
                           plugin.version,
                           ", ".join([v.short_str() for v in
                                      sorted(compatible_versions)]))
+                log.info(f"[pluginmanager] Successfully loaded plugin: {plugin.name} (version {plugin.version})")
                 plugin.compatible = True
                 setattr(picard.plugins, name, plugin_module)
                 if existing_plugin:
                     self.plugins[existing_plugin_index] = plugin
                 else:
                     self.plugins.append(plugin)
             else:
+                log.error(f"[pluginmanager] Plugin {plugin.name} is not compatible with API version(s) {getattr(plugin_module, 'PLUGIN_API_VERSIONS', None)}. Picard supports: {picard.api_versions_tuple}")
```

Comment:
The `plugin_error` function already logs an error. This line should be removed, and if a better error message is needed then a change to the following line should be made.